docs/src/IconButton.doc.js

```diff
@@ -1,419 +1,862 @@
+          onDismiss={() => setOpen(false)}
+        >
+          <Dropdown.Section label="Create">
+            <Dropdown.Item
+              onSelect={onSelect}
+              option={{ value: 'Pin', label: 'Pin' }}
+              selected={selected}
+            />
+            <Dropdown.Item
+              onSelect={onSelect}
+              option={{ value: 'Story Pin', label: 'Story Pin' }}
+              selected={selected}
+            />
+          </Dropdown.Section>
+        </Dropdown>
+      )}
+    </Flex>
+  );
+}
+      `}
+      />
+      <MainSection.Subsection
+        title="Keyboard interaction"
+        description={`
+The default behaviour for IconButton is to be focusable in sequential keyboard navigation in the order defined by the document's source order.
+
+Use \`tabIndex\` to remove IconButton from the sequential keyboard navigation to improve accessibility. The example below shows a common use case when two consecutive and visually different elements perform the same action. One of them, in this case IconButton, can be removed from keyboard navigation to prevent screen readers from announcing the same interaction twice.
+If IconButton is disabled, it's also unreachable from keyboard navigation.`}
+      />
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+function Example(props) {
+  return (
+    <Flex gap={2}>
+      <Avatar name="James Jones" src="https://i.ibb.co/2Fc00R3/james.jpg" size="md" />
+      <Flex gap={2} alignItems="center">
+        <Text inline weight="bold">
+          <Link
+            accessibilityLabel="Open the setting page"
+            target="blank"
+            inline
+            href="https://www.pinterest.com/settings/"
+          >
+            James Jones
+          </Link>
+        </Text>
+        <IconButton
+          accessibilityLabel="Open the setting page"
+          href="https://www.pinterest.com/settings/"
+          icon="edit"
+          role="link"
+          size="xs"
+          tabIndex="-1"
+          target="blank"
+        />
+      </Flex>
+    </Flex>
   );
 }
 `}
-  />,
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
+card(<MainSection name="Localization" description="Be sure to localize `accessibilityLabel`." />);
+
 card(
-  <Example
-    name="Icon"
-    id="icon"
-    defaultCode={`<Flex gap={2}>
-  <Tooltip text="Built-in Gestalt Icon">
-    <IconButton
-      accessibilityLabel="Add icon"
-      icon="directional-arrow-right"
-    />
-  </Tooltip>
-  <Tooltip text="Custom Icon">
-    <IconButton
-      accessibilityLabel="Open edit modal to edit Board"
-      dangerouslySetSvgPath={{ __path: 'M23 5v14a4 4 0 0 1-4 4H5a4 4 0 0 1-4-4v-5.5h10.258l-1.94 1.939a1.5 1.5 0 0 0 2.121 2.122L17 12l-5.561-5.561a1.501 1.501 0 0 0-2.121 2.122l1.94 1.939H1V5a4 4 0 0 1 4-4h14a4 4 0 0 1 4 4'}}
-    />
-  </Tooltip>
-</Flex>
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      title="Role"
```

Comment:
Could you split out some of these details in the description of the cards themselves, like this example: https://gestalt.netlify.app/Accessibility#Available-Hooks

Reply:
DONE. Good point